src/jdk.httpserver/share/classes/com/sun/net/httpserver/BasicAuthenticator.java

```diff
@@ -130,14 +131,14 @@ private void setAuthHeader(HttpExchange t) {
     }
 
     /**
-     * called for each incoming request to verify the
+     * Called for each incoming request to verify the
      * given name and password in the context of this
      * Authenticator's realm. Any caching of credentials
```

Comment:
Maybe remove the uppercase in "this Authenticator's realm"

Reply:
Change made in https://github.com/openjdk/jdk/pull/301/commits/5d9004612a60da62af3b30502ef2a1d8dfce2826